TaskHero/EditStepCell.swift

```diff
@@ -24,6 +24,8 @@ class EditStepCell: UITableViewCell {
     
     var delegate: EditStepCellDelegate?
     
+    var isTaskDetailFlow:Bool = false
```

Comment:
Just for styling guide, I think preferred is "var isTaskDetailFlow: Bool = false", and even the Bool isn't necessary.